### PR TITLE
His peer almighty

### PR DESCRIPTION
**Pull request by yossef-khaled**

### CODE REVIEW

File: content/post/his-peer-almighty.ar.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
++++
+title = "ترادف خَلقا، تجلى شأنا."
+date = "2022-12-03"
+author = "يوسف خالد"
+cover = "img/his-peer-almighty.jpg"
+CoverCaption = "Jessica Roux"
+images = ['img/his-peer-almighty.jpg']
++++
+`
+مَثَلُك الأعلى ، وموجّهك. قدوتك ، ومعلّمك. أكثرهم علماً، لأجله تفنى. أكملهم صواباً، فوقهم يعلى. بشرٌ كالبشر، ولكنه أسمى.
```
Comment (mhashim6) [inline content/post/his-peer-almighty.ar.md]:
```suggestion
مَثَلُك الأعلى ، وموجّهك. قدوتك ، ومعلّمك. أكثرهم علماً، لأجله تفنى. أكملهم صواباً، فوقهم يعلو. بشرٌ كالبشر، ولكنه أسمى.
```
I know it breaks your rhyme. 
How about you work with "أعلى" instead?
Or change previous sentences to support "يعلو"
Or if it doesn't bother you you can leave it with "يعلو" without further modifications.

File: content/post/his-peer-almighty.ar.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+بيده الأمر كله، أن يرسلك بعيداً، أو يجذبك إليه. يمسك أبواب الجنان، أمّا الجحيم فعلى مصرعيه. بشرٌ كالبشر، ولكنك مكنّته عليك.
```
Comment (mhashim6) [inline content/post/his-peer-almighty.ar.md]:
```suggestion
بيده الأمر كله، أن ينفيك بعيداً، أو يقربك إليه. يمسك أبواب الجنان، أمّا الجحيم فعلى مصراعيه. بشرٌ كالبشر، ولكنك مكنّته عليك.
```

File: content/post/his-peer-almighty.ar.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+يكذب و يسرق. جبالا من الذنوب في عنُقه، فاشهدوا سرّه. يغتصب و يقتل، غيرَ مبالٍ بذنبه. بشرٌ كالبشر، و لازلت بأثره.
+
+الوسيط، لتلك الجنان. لن تعبرها حتى يسمح لك. ليس وفق قانون، ولا حق. فلتؤمن به لذاتِه، و إلا فالنار أحق بك. بَشرٌ كالبشر، وسيقبع معك بقبرك.
```
Comment (mhashim6) [inline content/post/his-peer-almighty.ar.md]:
```suggestion
الوسيط، لتلك الجنان. لن تعبرها حتى يسمح لك. ليس وفق قانون، ولا حق. فلتؤمن به لذاتِه، و إلا فالنار بك أحق. بَشرٌ كالبشر، وسيقبع معك بقبرك.
```

File: content/post/his-peer-almighty.ar.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+وآخر، رسولٌ لكلمات. المصطفى، الأمين صادق الكلمات. مخبراً بالحق الإلهي، الكلمات التّامات. ماهو إلا رسول، لمجموعة من أناس ليسوا سعداء.
```
Comment (mhashim6) [inline content/post/his-peer-almighty.ar.md]: unhappy folk in this context means "البشر"
ايه رأيك نغيرها ازاي؟

File: content/post/his-peer-almighty.ar.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+مَثَلُك الأعلى ، وموجّهك. قدوتك ، ومعلّمك. أكثرهم علماً، لأجله تفنى. أكملهم صواباً، فوقهم يعلى. بشرٌ كالبشر، ولكنه أسمى.
+
+بيده الأمر كله، أن يرسلك بعيداً، أو يجذبك إليه. يمسك أبواب الجنان، أمّا الجحيم فعلى مصرعيه. بشرٌ كالبشر، ولكنك مكنّته عليك.
+
+يكذب و يسرق. جبالا من الذنوب في عنُقه، فاشهدوا سرّه. يغتصب و يقتل، غيرَ مبالٍ بذنبه. بشرٌ كالبشر، و لازلت بأثره.
+
+الوسيط، لتلك الجنان. لن تعبرها حتى يسمح لك. ليس وفق قانون، ولا حق. فلتؤمن به لذاتِه، و إلا فالنار أحق بك. بَشرٌ كالبشر، وسيقبع معك بقبرك.
+
+وآخر، رسولٌ لكلمات. المصطفى، الأمين صادق الكلمات. مخبراً بالحق الإلهي، الكلمات التّامات. ماهو إلا رسول، لمجموعة من أناس ليسوا سعداء. 
+لا يحكم بجَهده ولا بهواه، ولم يطلب من الذهب قنطاراً ولا باغيا لسلطانٍ أو جاه. لم يدّعي أنه إله، حاشاه. وليس للموافقة أو القَبول سبيل في "إن أراد ملكاً ملّكناه".
```
Comment (mhashim6) [inline content/post/his-peer-almighty.ar.md]:
```suggestion
لا يحكم بجَهده ولا بهواه، لا يطلب من الذهب قنطاراً ولا باغيا لسلطانٍ أو جاه. لا يدّعي أنه إله، حاشاه. وليس للموافقة أو القَبول سبيل في "إن أراد ملكاً ملّكناه".
```

This part is meant to represent all prophets.
لذلك "لا" اوقع من "لم" لاننا لا نتكلم بالضرورة عن حادثة واحدة بعينها.
ومع ذلك التوضيح على رسول الله محمد صلى الله عليه وسلم هو رائع وعبقري:
```
القَبول سبيل في "إن أراد ملكاً ملّكناه".
```

File: content/post/his-peer-almighty.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+The middle man, your gateway to Heavens. You shall not pass, unless he lets you in. Never appointed, or of a divine will. Accept him, lest you’re thrown in hell. A human, in the same grave he dwells.
+
+Another, delivers the word. The chosen one, an earthly lord. Speaks the truth, divine, and good. He’s but a messenger, to unhappy folk. He is no Judge, he acquires no gold. Seeks no control, nor claims to be God. Human, also. The only one to follow and uphold.
+
+> _Say, ˹O Prophet,˺ “I am only a man like you, ˹but˺ it has been revealed to me that your God is only One God. So whoever hopes for the meeting with their Lord, let them do good deeds and associate none in the worship of their Lord.”_
+> **[Chapter 18 (The Cave)](https://quran.com/18?startingVerse=110)**
+
+> _They have taken their rabbis and monks as well as the Messiah, son of Mary, as lords besides Allah, even though they were commanded to worship none but One God. There is no god ˹worthy of worship˺ except Him. Glorified is He above what they associate ˹with Him˺!_
+> **[Chapter 9 (Atonement)](https://quran.com/9?startingVerse=31)**
```
Comment (mhashim6) [inline content/post/his-peer-almighty.md]:
```suggestion
> **[Chapter 9 (Atonement)](https://quran.com/9/31)**
```

File: content/post/his-peer-almighty.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+He has all control, to cast you away, or lore you before. He keeps the gates of Heavens, and to Hell he leaves the door. A human, yet you make him more.
+
+He lies, he robs. Mountains of sins he holds—a secret to behold. He rapes, he kills. He cares not about his sins. A human, yet you follow him still.
+
+The middle man, your gateway to Heavens. You shall not pass, unless he lets you in. Never appointed, or of a divine will. Accept him, lest you’re thrown in hell. A human, in the same grave he dwells.
+
+Another, delivers the word. The chosen one, an earthly lord. Speaks the truth, divine, and good. He’s but a messenger, to unhappy folk. He is no Judge, he acquires no gold. Seeks no control, nor claims to be God. Human, also. The only one to follow and uphold.
+
+> _Say, ˹O Prophet,˺ “I am only a man like you, ˹but˺ it has been revealed to me that your God is only One God. So whoever hopes for the meeting with their Lord, let them do good deeds and associate none in the worship of their Lord.”_
+> **[Chapter 18 (The Cave)](https://quran.com/18?startingVerse=110)**
```
Comment (mhashim6) [inline content/post/his-peer-almighty.md]:
```suggestion
> **[Chapter 18 (The Cave)](https://quran.com/18/110)**
```

File: content/post/his-peer-almighty.ar.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+
+> _قُلْ إِنَّمَآ أَنَا۠ بَشَرٌۭ مِّثْلُكُمْ يُوحَىٰٓ إِلَىَّ أَنَّمَآ إِلَـٰهُكُمْ إِلَـٰهٌۭ وَٰحِدٌۭ ۖ فَمَن كَانَ يَرْجُوا۟ لِقَآءَ رَبِّهِۦ فَلْيَعْمَلْ عَمَلًۭا صَـٰلِحًۭا وَلَا يُشْرِكْ بِعِبَادَةِ رَبِّهِۦٓ أَحَدًۢا_\
+> **[سورة الكهف - آية ١١٠](https://quran.com/18?startingVerse=110)**
```
Comment (mhashim6) [inline content/post/his-peer-almighty.ar.md]:
```suggestion
> **[سورة الكهف - آية ١١٠](https://quran.com/18/110)**
```
My bad

File: content/post/his-peer-almighty.ar.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+> _۞ يَـٰٓأَيُّهَا ٱلَّذِينَ ءَامَنُوٓا۟ إِنَّ كَثِيرًۭا مِّنَ ٱلْأَحْبَارِ وَٱلرُّهْبَانِ لَيَأْكُلُونَ أَمْوَٰلَ ٱلنَّاسِ بِٱلْبَـٰطِلِ وَيَصُدُّونَ عَن سَبِيلِ ٱللَّهِ ۗ وَٱلَّذِينَ يَكْنِزُونَ ٱلذَّهَبَ وَٱلْفِضَّةَ وَلَا يُنفِقُونَهَا فِى سَبِيلِ ٱللَّهِ فَبَشِّرْهُم بِعَذَابٍ أَلِيمٍۢ_\
```
Comment (mhashim6) [inline content/post/his-peer-almighty.ar.md]:
```suggestion
> _ٱتَّخَذُوٓا۟ أَحْبَارَهُمْ وَرُهْبَـٰنَهُمْ أَرْبَابًۭا مِّن دُونِ ٱللَّهِ وَٱلْمَسِيحَ ٱبْنَ مَرْيَمَ وَمَآ أُمِرُوٓا۟ إِلَّا لِيَعْبُدُوٓا۟ إِلَـٰهًۭا وَٰحِدًۭا ۖ لَّآ إِلَـٰهَ إِلَّا هُوَ ۚ سُبْحَـٰنَهُۥ عَمَّا يُشْرِكُونَ_\
```
My bad it was a wrong link to the verse

**Review by mhashim6 — COMMENTED** [review]
ما شاء الله والله ترجمة عبقرية. بلحنها الخاص وتفردها وبلاغتها.
تعديلات صغيرة بس 🙏🏻

File: content/post/his-peer-almighty.ar.md (hunk 1 above)
Comment (yossef-khaled) [reply content/post/his-peer-almighty.ar.md]: Yeah, right. I guess يعلو is the right one as it is singular.

File: content/post/his-peer-almighty.ar.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+> **[سورة التوبة - آية ٣٤](https://quran.com/9?startingVerse=31)**
+
+>_حَدَّثَنَا عَبْدُ اللَّهِ بْنُ مُحَمَّدٍ، قَالَ حَدَّثَنَا هَاشِمُ بْنُ الْقَاسِمِ، قَالَ حَدَّثَنَا شَيْبَانُ أَبُو مُعَاوِيَةَ، عَنْ زِيَادِ بْنِ عِلاَقَةَ، عَنِ الْمُغِيرَةِ بْنِ شُعْبَةَ، قَالَ كَسَفَتِ الشَّمْسُ عَلَى عَهْدِ رَسُولِ اللَّهِ صلى الله عليه وسلم يَوْمَ مَاتَ إِبْرَاهِيمُ، فَقَالَ النَّاسُ كَسَفَتِ الشَّمْسُ لِمَوْتِ إِبْرَاهِيمَ. فَقَالَ رَسُولُ اللَّهِ صلى الله عليه وسلم **" إِنَّ الشَّمْسَ وَالْقَمَرَ لاَ يَنْكَسِفَانِ لِمَوْتِ أَحَدٍ وَلاَ لِحَيَاتِهِ، فَإِذَا رَأَيْتُمْ فَصَلُّوا وَادْعُوا اللَّهَ ".**_
```
Comment (mhashim6) [inline content/post/his-peer-almighty.ar.md]: Please remove the "عنعنة" And add a link instead 🙏🏻

File: content/post/his-peer-almighty.ar.md (hunk 3 above)
Comment (yossef-khaled) [reply content/post/his-peer-almighty.ar.md]: I thought I did it as you wrote it. Weird.

Comment (mhashim6) [reply content/post/his-peer-almighty.ar.md]: @yossef-khaled I guessed so والله :D

File: content/post/his-peer-almighty.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+He has all control, to cast you away, or lore you before. He keeps the gates of Heavens, and to Hell he leaves the door. A human, yet you make him more.
+
+He lies, he robs. Mountains of sins he holds—a secret to behold. He rapes, he kills. He cares not about his sins. A human, yet you follow him still.
+
+The middle man, your gateway to Heavens. You shall not pass, unless he lets you in. Never appointed, or of a divine will. Accept him, lest you’re thrown in hell. A human, in the same grave he dwells.
+
+Another, delivers the word. The chosen one, an earthly lord. Speaks the truth, divine, and good. He’s but a messenger, to unhappy folk. He is no Judge, he acquires no gold. Seeks no control, nor claims to be God. Human, also. The only one to follow and uphold.
+
+> _Say, ˹O Prophet,˺ “I am only a man like you, ˹but˺ it has been revealed to me that your God is only One God. So whoever hopes for the meeting with their Lord, let them do good deeds and associate none in the worship of their Lord.”_
+> **[Chapter 18 (The Cave)](https://quran.com/18/110)**
```
Comment (mhashim6) [inline content/post/his-peer-almighty.md]:
```suggestion
> _Say, ˹O Prophet,˺ “I am only a man like you, ˹but˺ it has been revealed to me that your God is only One God. So whoever hopes for the meeting with their Lord, let them do good deeds and associate none in the worship of their Lord.”_ \
> —**[Chapter 18 (The Cave)](https://quran.com/18/110)**
```

File: content/post/his-peer-almighty.md [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+
+The middle man, your gateway to Heavens. You shall not pass, unless he lets you in. Never appointed, or of a divine will. Accept him, lest you’re thrown in hell. A human, in the same grave he dwells.
+
+Another, delivers the word. The chosen one, an earthly lord. Speaks the truth, divine, and good. He’s but a messenger, to unhappy folk. He is no Judge, he acquires no gold. Seeks no control, nor claims to be God. Human, also. The only one to follow and uphold.
+
+> _Say, ˹O Prophet,˺ “I am only a man like you, ˹but˺ it has been revealed to me that your God is only One God. So whoever hopes for the meeting with their Lord, let them do good deeds and associate none in the worship of their Lord.”_ \
+> —**[Chapter 18 (The Cave)](https://quran.com/18/110)**
+
+> _They have taken their rabbis and monks as well as the Messiah, son of Mary, as lords besides Allah, even though they were commanded to worship none but One God. There is no god ˹worthy of worship˺ except Him. Glorified is He above what they associate ˹with Him˺!_
+> **[Chapter 9 (Atonement)](https://quran.com/9/31)**
```
Comment (mhashim6) [inline content/post/his-peer-almighty.md]:
```suggestion
> _They have taken their rabbis and monks as well as the Messiah, son of Mary, as lords besides Allah, even though they were commanded to worship none but One God. There is no god ˹worthy of worship˺ except Him. Glorified is He above what they associate ˹with Him˺!_ \
> —**[Chapter 9 (Atonement)](https://quran.com/9/31)**
```

**Review by mhashim6 — APPROVED** [review]

**Pull request merged**